website/integrations/services/home-assistant/index.md

```diff
@@ -16,6 +16,9 @@ sidebar_label: Home Assistant
 :::caution
 You might run into CSRF errors, this is caused by a technology Home-assistant uses and not authentik, see [this GitHub issue](https://github.com/goauthentik/authentik/issues/884#issuecomment-851542477).
 :::
+:::caution
+Only `/auth` needs to proxied (excluding `/auth/token`), see [this GitHub issue](https://github.com/BeryJu/hass-auth-header/issues/212).
```

Comment:
Thanks for this important addition! Can you explain a little more what you mean by "excluding `/auth/token` please?

Reply:
Thanks @sbogomolov I appreciate that clarification. This is an area I am trying to learn more about, but not fully there yet, lol.

We do need to provide the reader with info on *how* to accomplish this task of "proxying the prefixes".  Let's add some basic info about how/where to do this in authentik. Let's also add the an example list of prefixes (taken from the Issue [#1842 ](https://github.com/authelia/authelia/issues/1842#issuecomment-977695269). Though, interestingly, that list includes `/auth/token`. Hmmm...

Reply:
```suggestion
You can use the **Unauthenticated Paths** setting in the proxy provider to define regular expressions for which authentication via a proxy is not required. Typically, only `/auth` prefixes needs to proxied (excluding `/auth/token`). 

<img width="1219" alt="Screenshot 2024-12-03 at 2 48 49 PM" src="https://github.com/user-attachments/assets/a0508c4c-4aa2-436b-a811-9b5655cceffd">

For more information refer to, see [GitHub issue #212](https://github.com/BeryJu/hass-auth-header/issues/212).
```

Reply:
```suggestion
You can use the **Unauthenticated Paths** setting in the proxy provider to define regular expressions for which authentication via a proxy is not required. Typically, only `/auth` prefixes needs to proxied (excluding `/auth/token`). 

<img width="1219" alt="Screenshot 2024-12-03 at 2 53 31 PM" src="https://github.com/user-attachments/assets/14fa005a-257a-4239-84d8-a406b3517487">

For more information refer to, see [GitHub issue #212](https://github.com/BeryJu/hass-auth-header/issues/212).

Reply:
Please review carefully, to verify that my suggestion is accurate (using the Unauthenticated Paths field to add any prefixes that do not need to be proxied) and my attempt at adding escapes to the wildcards in the prefixes. @BeryJu can you also please look over this suggested wording?

Reply:
To be honest, I managed to do this via Traefik's Ingress Route setup. I did not know I can do this in authentik :) Here is what I did:
```yaml
  routes:
  - kind: Rule
    match: Host(`ha.domain.tld`)
    services:
    - name: ha
      port: http
  - kind: Rule
    match: Host(`ha.domain.tld`) && PathPrefix(`/auth`)
    middlewares:
    - name: authentik
      namespace: authentik
    services:
    - name: ha
      port: http
  - kind: Rule
    match: Host(`ha.domain.tld`) && PathPrefix(`/auth/token`)
    services:
    - name: ha
      port: http
  - kind: Rule
    match: Host(`ha.domain.tld`) && PathPrefix(`/outpost.goauthentik.io/`)
    services:
    - name: authentik-server
      namespace: authentik
      port: http
      nativeLB: true

```

Reply:
I have tried adding this list to Unauthenticated Paths and removing my workaround in IngressRoute:
```
^/api.*
^/auth/token.*
^/.external_auth=.
^/service_worker.js
^/static.*
^/local.*
^/hacsfiles.*
^/frontend_latest.*
```
It did not work. I see that requests to `/api/...` return `401 Unauthorized`.

Reply:
Ah, okay, thanks very much for that feedback @sbogomolov. I hadn't realized you'd configured it in Traefik, makes perfect sense. 

I think for now it's best if we go back to your original wording, but please clarify that this can be set in a reverse proxy (such as Traefik) OR in authentik (on the Proxy Provider), and we get this PR merged. I'll ask Engineering to test what you found; if you have any info to share you could open an Issue and describe what happened when you tried to use the Unauthenticated Paths field in authentik. Thanks so much for this contribution, and for your patience with working out the details.

Reply:
I have updated the wording. Let me know what you think.